.github/workflows/tests.yml

```diff
@@ -26,7 +26,11 @@ jobs:
         shell: bash -l {0}
         run: |
           mamba install --file requirements.txt pre-commit coverage coveralls mock pytest pytest-cov pip ${CONDA_PKGS} --yes --quiet
-          python -m pip install -e .
+          # get pyproject.toml out of the way so setuptools doesn't complain
+          mv pyproject.toml pyproject.toml.bak
```

Comment:
What is setuptools complaining about?

Reply:
I guess we could also pin setuptools to an earlier (no clue which one) version, but same point as above. I just wanted a quick fix to unblock people affected by #1677

Reply:
Ok let's raise an issue to address this later (likely by migrating to `pyproject.toml`)

Reply:
Is this a versionneer issue? If so i e been moving my projects to miniver https://github.com/jbweston/miniver

Reply:
I thought so at first (hence the branch name), but it's unrelated. The warning we get right on top of the error just happens to be there.